### PR TITLE
Fix typo on docs/contributing

### PR DESCRIPTION
**Pull request by CyanBook**

### CODE REVIEW

**Comment by Kludex** [comment]
@CyanBook It's not a typo: https://dictionary.cambridge.org/us/dictionary/english/clash

**Comment by CyanBook** [comment]
Anyway, "clash" isn't the proper verb to use. "Crash" is better imo.

File: docs/en/docs/contributing.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ $ uvicorn tutorial001:app --reload
 
 </div>
 
-as Uvicorn by default will use the port `8000`, the documentation on port `8008` won't clash.
+as Uvicorn by default will use the port `8000`, the documentation on port `8008` won't crash.
```
Comment (Kludex) [inline docs/en/docs/contributing.md:262]: Based on your comment I've investigated and it looks like the most known term seems to be `overlap`. :thinking: 
```suggestion
as Uvicorn by default will use the port `8000`, the documentation on port `8008` won't overlap.
```

Comment (CyanBook) [reply docs/en/docs/contributing.md:262]: I agree with you. "overlap" is better than "crash"

**Review by ycd — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks for your interest! But I actually think "clash" would be more appropriate here, as it wouldn't be possible for applications to take the same port twice, ad the same time, and overlap. One of the two applications will crash, so I think the word "clash" conveys that message better.

Given that, I'll pass on this one. But thanks for the effort! :coffee:

**Pull request closed without merging**